R/clustering.R

```diff
@@ -13,6 +13,9 @@ NULL
 #' @importFrom pbapply pblapply
 #' @importFrom future.apply future_lapply
 #' @importFrom future nbrOfWorkers
+#' @importFrom igraph graph_from_adjacency_matrix
+#' @importFrom methods as
```

Comment:
Is this being used in `FindClusters.default` anywhere? I can't find any calls to `methods::as`

Reply:
Sorry, yes these have been removed.